frontend/lib/routes/meeting/EditMeetingForm.dart

```diff
@@ -0,0 +1,237 @@
+import 'package:flutter/material.dart';
+import 'package:frontend/models/meeting.dart';
+import 'package:frontend/routes/meeting/MeetingsNotifier.dart';
+import 'package:frontend/services/meetingService.dart';
+import 'package:intl/intl.dart';
+import 'package:provider/provider.dart';
+
+class EditMeetingForm extends StatefulWidget {
+  final Meeting meeting;
+  EditMeetingForm({Key? key, required this.meeting}) : super(key: key);
+
+  @override
+  _EditMeetingFormState createState() => _EditMeetingFormState();
+}
+
+var kinds = ["Event", "Team", "Company"];
```

Comment:
Minor thing: I think this can be a `const`, right?

Reply:
Yes, you're right :+1: Fixed :+1: